src/sentry/static/sentry/app/views/settings/components/forms/formField/index.jsx

```diff
@@ -143,7 +144,10 @@ class FormField extends React.Component {
    * Set field's hover state and propagate callbacks
    */
   handleHover = (mouseOver, ...args) => {
-    let {name, onMouseOver, onMouseOut} = this.props;
+    let {name, onMouseOver, onMouseOut, noHover} = this.props;
+
+    if (noHover) return;
```

Comment:
Could `noHover` be default? Or do many of the fields have hover states?

Reply:
I completely removed this, instead removed handling of hover state from react state and moved it into our styled component